avocado/core/job.py

```diff
@@ -319,10 +319,14 @@ def _log_avocado_version():
         job_log = _TEST_LOGGER
         job_log.info('Avocado version: %s', version.VERSION)
         if os.path.exists('.git') and os.path.exists('avocado.spec'):
-            cmd = "git show --summary --pretty='%H' | head -1"
-            status, top_commit = commands.getstatusoutput(cmd)
+            cmd = "git show --summary --pretty='%H'"
+            result = process.run(cmd, ignore_status=True)
```

Comment:
Oh, sorry, I approved the PR before seeing your comment. Let's wait @clebergnu's answer on that (regardless the fact that I disagree with the suggestion).